README.md

```diff
@@ -193,9 +202,15 @@ roslaunch nvidia_gmsl_driver_ros nvidia_gmsl_driver_ros.launch
 
 | Parameter        |                           Default |                               Comment |
 |------------------|----------------------------------:|--------------------------------------:|
-| `config_path`    | `$(dirname)/../config/ports.yaml` |               Path to the config file |
-| `calib_dir_path` |            `$(dirname)/../calib/` |   Path to the camera calibration file |
-| `framerate`      |                              `30` |                      Output framerate |
-| `verbose`        |                           `False` |                Enables verbose output | 
+| `config_path`    | `$(dirname)/../config/ports.yaml` |              Path to the config file. |
+| `calib_dir_path` |            `$(dirname)/../calib/` |  Path to the camera calibration file. |
+| `framerate`      |                              `30` |                     Output framerate. |
+| `verbose`        |                           `False` |               Enables verbose output. | 
 | `encoder`        |                             `jpg` |            Encoder. (`jpg` or `h264`) | 
 | `h264_bitrate`   |                         `8000000` | h264 output bitrate (Minimum `30000`) | 
```

Comment:
Needs to be updated.

Reply:
Can you elaborate what is missing here?

Reply:
The parameter `h264_bitrate` has been renamed `bitrate`